### PR TITLE
fixes : 5021

### PR DESCRIPTION
**Pull request by tsl143**
Suggesting default full width of biography and resize only vertically
Fixes : #5021

### CODE REVIEW

File: src/amo/components/UserProfileEdit/styles.scss [hunk 1]
```diff
@@ -86,6 +86,13 @@ $font-size-aside: 10px;
   }
 }
 
+.UserProfileEdit textarea {
```
Comment (willdurand) [inline src/amo/components/UserProfileEdit/styles.scss]: Could you use `.UserProfileEdit-biography` instead?

Comment (tsl143) [reply src/amo/components/UserProfileEdit/styles.scss]: Updated!

**Comment by willdurand** [comment]
So that does not work anymore because the rule is overridden by another one. I think we could set the `textarea` width to `100%` all the time and `resize: vertical` too. What do you think?

```diff
diff --git a/src/amo/components/UserProfileEdit/styles.scss b/src/amo/components/UserProfileEdit/styles.scss
index 1cd36e4..b91d395 100644
--- a/src/amo/components/UserProfileEdit/styles.scss
+++ b/src/amo/components/UserProfileEdit/styles.scss
@@ -80,7 +80,9 @@ $font-size-aside: 10px;
     background: $grey-20;
     cursor: not-allowed;
   }
+}
 
+.UserProfileEdit input {
   @include respond-to(large) {
     width: 60%;
   }
@@ -95,11 +97,7 @@ $font-size-aside: 10px;
 
 .UserProfileEdit-biography {
   min-height: 3rem;
-
-  @include respond-to(large) {
-    resize: vertical;
-    width: 100%;
-  }
+  resize: vertical;
 }
 
 .UserProfileEdit-buttons-wrapper {
```

**Comment by tsl143** [comment]
Sounds good, I am updating PR, I cant test locally, (I dont have the setup on my office machine :P ) hoping this fixes.

**Comment by willdurand** [comment]
Thanks for your PR!

**Pull request merged**